coops-ui/src/views/Dashboard.vue

```diff
@@ -91,7 +137,10 @@ export default {
       todoListFilings: [],
       refreshTimer: null,
       checkFilingStatusCount: 0,
-      inProcessFiling: null
+      inProcessFiling: null,
+      coaPending: false,
+      effectiveDate: null,
+      FilingStatus
```

Comment:
Please add a comment for this one (same as in AnnualReport:297) cause it looks weird among the other 'real' data properties ;)

Reply:
Done and done, will do, thanks!